src/iop/lut3d.c

```diff
@@ -842,7 +842,7 @@ uint16_t calculate_clut_cube(const char *const filepath, float **clut)
   }
   if(out_of_range_nb)
   {
-    fprintf(stderr, "[lut3d] warning - %d values out of range [0,1]\n", out_of_range_nb);
+    fprintf(stderr, "[lut3d] warning - %u values out of range [0,1]\n", out_of_range_nb);
```

Comment:
Since we're changing the code anyway, should this be switched to dt_print?

Reply:
A good idea, `dt_print(DT_DEBUG_ALWAYS,  "[lut3d]...`

Reply:
Done for all the output for the two modules in question.